website/static/js/registrationModal.js

```diff
@@ -47,21 +45,29 @@ var RegistrationViewModel = function(confirm, prompts, validator) {
         }
     );
     self.embargoEndDate = ko.computed(function() {
-        return new Date(self.pikaday());
+        return moment(new Date(self.pikaday())).subtract(utcOffset, 'minutes');
     });
 
+    self.minimumTimeValidation = function (x, y, embargoLocalDateTime) {
+        var minEmbargoMoment = getMinimumDate(embargoLocalDateTime),
+            endEmbargoMoment = self.embargoEndDate();
+        return minEmbargoMoment.isBefore(endEmbargoMoment);
+    };
+
+    self.maximumTimeValidation = function (x, y, embargoLocalDateTime) {
+        var maxEmbargoMoment = getMaximumDate(embargoLocalDateTime),
+            endEmbargoMoment = self.embargoEndDate();
+        return maxEmbargoMoment.isAfter(endEmbargoMoment);
+    };
+
     var validation = [{
-        validator: function() {
-            var endEmbargoDateTimestamp = self.embargoEndDate().getTime();
-            return endEmbargoDateTimestamp > TWO_DAYS_FROM_TODAY_TIMESTAMP;
-        },
+        validator: self.minimumTimeValidation
+        ,
```

Comment:
I imagine this was one of the hint errors, but move this comma up to the previous line.